aics_pipeline_uploaders/util/celigo.py

```diff
@@ -1,44 +1,38 @@
 from datetime import datetime
+
 import requests
 
 SERVICE_MMS = "mms"
 MMS_RESOURCE_PLATE_INFO = "plate_info"
-MMS_STG_URL = 'http://stg-aics-api.corp.alleninstitute.org/metadata-management-service/1.0/plate/query?barcode='
-MMS_PROD_URL = 'http://aics-api.corp.alleninstitute.org/metadata-management-service/1.0/plate/query?barcode='
+MMS_STG_URL = "http://stg-aics-api.corp.alleninstitute.org/metadata-management-service/1.0/plate/query?barcode="
+MMS_PROD_URL = "http://aics-api.corp.alleninstitute.org/metadata-management-service/1.0/plate/query?barcode="
 ENV_SERVICE_MAP = {
-    "stg": {
-        SERVICE_MMS: {
-            MMS_RESOURCE_PLATE_INFO: MMS_STG_URL
-        }
-    },
-    "prod": {
-        SERVICE_MMS: {
-            MMS_RESOURCE_PLATE_INFO: MMS_PROD_URL
-        }
-    }
+    "stg": {SERVICE_MMS: {MMS_RESOURCE_PLATE_INFO: MMS_STG_URL}},
+    "prod": {SERVICE_MMS: {MMS_RESOURCE_PLATE_INFO: MMS_PROD_URL}},
 }
 
 
-class CeligoUtil():
-
+class CeligoUtil:
     def __init__(self, env: str = "stg"):
         self.env = env
 
     def lookup_well_id(self, plate_barcode, well_name):
-        url = ENV_SERVICE_MAP[self.env][SERVICE_MMS][MMS_RESOURCE_PLATE_INFO] + str(plate_barcode)
+        url = ENV_SERVICE_MAP[self.env][SERVICE_MMS][MMS_RESOURCE_PLATE_INFO] + str(
+            plate_barcode
+        )
         well_info_response = requests.get(url)
         well_info = well_info_response.json()
-        if len(well_info['data'][0]) > 1:
```

Comment:
Noticed that len(well_info['data'][0]) is the length of an individual imaging sessions info rather than the number of imaging sessions. Adjusted by removing [0].